Missing argument for CookieStore::new()

I am currently trying out the 3.0.0-rc1, but when trying to compile i get the following error: 
```error[E0061]: this function takes 1 argument but 0 arguments were supplied
   --> /home/user/.cargo/git/checkouts/ureq-cb1f1b767b8d67d8/014b8f6/src/cookies.rs:147:31
    |
147 |             inner: Mutex::new(CookieStore::new()),
    |                               ^^^^^^^^^^^^^^^^-- argument #1 of type `std::option::Option<publicsuffix::List>` is missing
    |
note: associated function defined here
   --> /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cookie_store-0.21.0/src/cookie_store.rs:495:12
    |
495 |     pub fn new(
    |            ^^^
help: provide the argument
    |
147 |             inner: Mutex::new(CookieStore::new(/* std::option::Option<publicsuffix::List> */)),
    |                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
This happens on the rc, as well as on the main git branch. This could totaly be some kind of issue on my side, but I thought I'd let you know. Thanks in advance for your response!
## Edit
When testing this in isolation i couldn't replicate this issue, even when using the same features. Deleting the Cargo.lock also didnt resolve this. 
The repo im having this issue with is https://github.com/jullanggit/tam_intranet

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,6 @@
 # Unreleased
 
+ * Fix featue flag `public_suffix` for CookieStore (#1063)
  * Improve doc for 10MB limit (#1061)
 
 # 3.0.11

--- a/src/cookies.rs
+++ b/src/cookies.rs
@@ -1,5 +1,6 @@
 use std::borrow::Cow;
 use std::fmt;
+use std::iter;
 use std::sync::{Mutex, MutexGuard};
 
 use cookie_store::CookieStore;
@@ -168,10 +169,20 @@ impl<'a> CookieJar<'a> {
     pub fn release(self) {}
 }
 
+// CookieStore::new() changes parameters depending on feature flag "public_suffix".
+// That means if a user enables public_suffix for CookieStore through diamond dependency,
+// we start having compilation errors un ureq.
+//
+// This workaround instantiates a CookieStore in a way that does not change with flags.
+fn instantiate_cookie_store() -> CookieStore {
+    let i = iter::empty::<Result<cookie_store::Cookie<'static>, &str>>();
+    CookieStore::from_cookies(i, true).unwrap()
+}
+
 impl SharedCookieJar {
     pub(crate) fn new() -> Self {
         SharedCookieJar {
-            inner: Mutex::new(CookieStore::new()),
+            inner: Mutex::new(instantiate_cookie_store()),
         }
     }
 

